packages/react-charts/src/components/ChartTooltip/examples/ChartTooltip.md

```diff
@@ -297,7 +297,7 @@ This demonstrates how to embed HTML within a tooltip. Combining cursor and voron
 
 ```js
 import React from 'react';
-import { Chart, ChartArea, ChartAxis, ChartCursorFlyout, ChartCursorTooltip, ChartGroup, ChartPoint, ChartThemeColor } from '@patternfly/react-charts';
+import { Chart, ChartArea, ChartAxis, ChartCursorFlyout, ChartCursorTooltip, ChartGroup, ChartPoint, ChartThemeColor, createContainer } from '@patternfly/react-charts';
```

Comment:
I think createContainer is an unnecessary unused import

Reply:
It's used on line 315. I noticed it because codesandbox fails to load the example without it.